src/api/lib/tasks/auto_annotate_models.rake

```diff
@@ -0,0 +1,34 @@
+# NOTE: only doing this in development as some production environments (Heroku)
+# NOTE: are sensitive to local FS writes, and besides -- it's just not proper
+# NOTE: to have a dev-mode tool do its thing in production.
+if Rails.env.development?
+  task :set_annotation_options do
+    # You can override any of these by setting an environment variable of the
+    # same name.
+    Annotate.set_defaults({
+      'position_in_routes'   => "before",
+      'position_in_class'    => "before",
```

Comment:
@hennevogel, I would prefer `position_in_class' => 'after'` to add the 'schema information' at the bottom instead of the beginning